helix-core/src/main/java/org/apache/helix/task/TaskUtil.java

```diff
@@ -789,14 +789,19 @@ protected static boolean removeJobsFromWorkflow(final HelixDataAccessor dataAcce
    */
   public static Set<String> getExpiredJobsFromCache(
       WorkflowControllerDataProvider workflowControllerDataProvider, WorkflowConfig workflowConfig,
-      WorkflowContext workflowContext) {
+      WorkflowContext workflowContext, HelixManager manager) {
```

Comment:
This is a public API change. If there is other place using this API, the compilation may fail.